<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OPS Design</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Motor Simple</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;RESULTADOS DE ANÁLISIS Y DISEÑO - [SECCIONES DE INICIALIZACIÓN]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Presentación de gráficas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonGraficaOneTime">
             <property name="text">
              <string>Pórtico</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" rowspan="4">
         <widget class="QTextBrowser" name="textBrowser_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Detalles específicos</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>M.C.E</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="toolButton_tbuni_one">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QToolButton" name="toolButton_tbcef_one">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="toolButton_tbdis_one">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Analisis</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Diseño</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Uniones</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="toolButton_tban_one">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>Información</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="2" column="1">
            <widget class="QToolButton" name="toolButton_visor_Ra">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Visor de Perfiles</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 229, 229);
color:withe;
border-style: outset;
border-width:1px;
border-radius:15px</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButtonAyuda_Ra">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(229, 229, 229);
color:withe;
border-style: outset;
border-width:0px;
border-radius:15px
</string>
                </property>
                <property name="text">
                 <string>Guía y Blog</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Motor de Optimización</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;RESULTADOS DE ANÁLISIS Y DISEÑO - [SECCIONES OPTIMIZADAS]&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Diseños representativos</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QRadioButton" name="radioButton_op2">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text">
              <string>Opción B</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButton_op1">
             <property name="text">
              <string>Opción A</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" rowspan="5">
         <widget class="QTextBrowser" name="textBrowser_3"/>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Presentación de gráficas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonGrafica_pesos">
             <property name="text">
              <string>Función Objetivo</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButtonGraficaIteration">
             <property name="text">
              <string>Pórtico</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Detalles específicos</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Analisis</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Uniones</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Diseño</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>M.C.E</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QToolButton" name="toolButton_tban_iter">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QToolButton" name="toolButton_tbdis_iter">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QToolButton" name="toolButton_tbuni_iter">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QToolButton" name="toolButton_tbcef_iter">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>Información</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="2" column="1">
            <widget class="QToolButton" name="toolButton_visor_Rb">
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Visor de Perfiles</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox_8">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(229, 229, 229);
color:withe;
border-style: outset;
border-width:1px;
border-radius:15px</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButtonAyuda_Rb">
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(229, 229, 229);
color:withe;
border-style: outset;
border-width:0px;
border-radius:15px
</string>
                </property>
                <property name="text">
                 <string>Guía y Blog</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>235</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Unidades: kgf - m - sec&lt;span style=&quot; color:#ffffff;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
